canonical-web-and-design/jaas-dashboard
# Populate controller table with model counts

## Done
Populate controller table with model counts

## QA
- Be in the [~yellow team](https://launchpad.net/~yellow/+members)
- Change the `baseControllerURL` in `public/config.js` to `jimm.staging.jujucharms.com`
https://github.com/canonical-web-and-design/jaas-dashboard/blob/411f115fbc422105c6ca026ba9a4200c0629610c/public/config.js#L6
- Create a model on the staging JAAS.
  - Register staging jaas via the CLI on your machine, then add a model or two.
- Run `./run`
- Open http://localhost:8036/controllers
- See that "models | machines | applications | units" columns in the table contain values.
